cli/package.json

```diff
@@ -14,44 +14,43 @@
   },
   "license": "SEE LICENSE IN LICENSE-*",
   "dependencies": {
-    "chalk": "~2.1.0",
-    "command-line-args": "^4.0.2",
-    "command-line-commands": "^2.0.0",
-    "command-line-usage": "^4.0.0",
+    "chalk": "~3.0.0",
+    "command-line-args": "^5.1.1",
+    "command-line-commands": "^3.0.1",
+    "command-line-usage": "^6.1.0",
     "git-config": "0.0.7",
-    "handlebars": "^4.1.0",
-    "inquirer": "^3.0.6",
-    "mkdirp": "^0.5.1",
+    "handlebars": "^4.7.3",
+    "inquirer": "^7.1.0",
     "quickly-copy-file": "^1.0.0",
-    "rimraf": "^2.6.1",
-    "rsvp": "^4.6.1",
-    "semver": "^5.1.0",
-    "toml": "^2.3.0",
+    "rimraf": "^3.0.2",
+    "rsvp": "^4.8.5",
+    "semver": "^7.1.3",
+    "toml": "^3.0.0",
     "ts-typed-json": "^0.2.2",
-    "validate-npm-package-license": "^3.0.1",
+    "validate-npm-package-license": "^3.0.4",
     "validate-npm-package-name": "^3.0.0"
   },
   "devDependencies": {
-    "@types/chai": "^4.1.3",
-    "@types/chalk": "^0.4.31",
-    "@types/es6-promise": "0.0.33",
-    "@types/inquirer": "0.0.35",
-    "@types/mkdirp": "^0.5.1",
-    "@types/mocha": "^5.2.0",
-    "@types/nexpect": "^0.4.29",
-    "@types/node": "^8.10.39",
-    "@types/rimraf": "^2.0.2",
-    "@types/rsvp": "^3.3.1",
-    "@types/semver": "^5.3.31",
-    "@types/tmp": "0.0.33",
-    "chai": "^4.1.2",
-    "mocha": "^5.2.0",
-    "nexpect": "^0.5.0",
-    "tmp": "0.0.33",
+    "@types/chai": "^4.2.11",
+    "@types/chalk": "^2.2.0",
+    "@types/es6-promise": "3.3.0",
+    "@types/inquirer": "6.5.0",
+    "@types/mkdirp": "^1.0.0",
+    "@types/mocha": "^7.0.2",
+    "@types/nexpect": "^0.4.30",
+    "@types/node": "^13.9.2",
+    "@types/rimraf": "^3.0.0",
+    "@types/rsvp": "^4.0.3",
+    "@types/semver": "^7.1.0",
+    "@types/tmp": "0.1.0",
+    "chai": "^4.2.0",
+    "mocha": "^7.1.1",
+    "nexpect": "^0.6.0",
+    "tmp": "0.1.0",
     "ts-dict": "^0.1.1",
-    "ts-node": "^3.0.2",
+    "ts-node": "^8.7.0",
     "ts-unknown": "^0.2.0",
```

Comment:
Should this be removed?

Reply:
Oh, this was my omission. I'll fix it. Thank you!